packages/forms/src/Form.js

```diff
@@ -23,10 +24,11 @@ const customWidgets = {
 	tabs: TabsWidget,
 	keyValue: KeyValueWidget,
 	multiSelectTag: MultiSelectTagWidget,
+	simpleTypeahead: SimpleTypeaheadWidget,
 };
 
 const customUiSchema = {
-	'ui:widget': ['toggle', 'tabs', 'keyValue', 'multiSelectTag'],
+	'ui:widget': ['toggle', 'tabs', 'keyValue', 'multiSelectTag', 'simpleTypeahead'],
```

Comment:
It could be nice if its name could match a known HTML5 element like `datalist` instead of `simpleTypeahead`